framework/SupervisedLearning/ARMA.py

```diff
@@ -716,17 +719,26 @@ def _generateARMASignal(self, model, numSamples=None,randEngine=None):
       numSamples =  len(self.pivotParameterValues)
     if randEngine is None:
       randEngine=self.randomEng
-    import statsmodels.api
-    hist = statsmodels.api.tsa.arma_generate_sample(ar = np.append(1., -model.arparams),
+    import statsmodels.tsa
+    hist = statsmodels.tsa.arima_process.arma_generate_sample(ar = np.append(1., -model.arparams),
                                                     ma = np.append(1., model.maparams),
                                                     nsample = numSamples,
                                                     distrvs = functools.partial(randomUtils.randomNormal,engine=randEngine),
                                        # functool.partial provide the random number generator as a function
                                        # with normal distribution and take engine as the positional arguments keywords.
-                                                    sigma = np.sqrt(model.sigma2),
+                                                    scale = np.sqrt(model.sigma2),
                                                     burnin = 2*max(self.P,self.Q)) # @epinas, 2018
+    #hist = statsmodels.tsa.arima_process.arma_generate_sample(ar = np.append(1., -model.arparams),
```

Comment:
Can this comment be removed? (Or should it be noted that it is for an older statsmodel?)

Reply:
I would appreciate leaving this here at least until the new changes can be tested in production a bit more, for reference.